main.go

```diff
@@ -226,6 +229,39 @@ func addEnvVars(ciProvider string) {
 	}
 }
 
+// addlFields adds an arbitrary set of fields provided by the end user
+func addlFields() {
+	locn := os.Getenv("BUILDEVENT_FILE")
+	if locn == "" {
+		return
+	}
+
+	data, err := ioutil.ReadFile(locn)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "unable to read %q: %v\n", locn, err)
+		return
+	}
+
+	err = logfmt.Unmarshal(
+		data,
+		logfmt.HandlerFunc(func(key, val []byte) error {
+			if f, err := strconv.ParseFloat(string(val), 64); err == nil {
+				libhoney.AddField(string(key), f)
+				return nil
+			}
+			if b, err := strconv.ParseBool("true"); err == nil {
```

Comment:
🤦‍♂ sorry I gave you a bad suggestion before.

```suggestion
			if b, err := strconv.ParseBool(string(val)); err == nil {
```

Reply:
Also on me... I shoulda looked more carefully before mashing the "commit" button!